src/Element.jl

```diff
@@ -24,3 +25,26 @@ n_elec_valence(attype::Element) = charge_ionic(attype)
 
 """Return the number of core electrons"""
 n_elec_core(attype::Element) = charge_nuclear(attype) - charge_ionic(attype)
+
+"""Radial local potential, in Fourier space: V(q) = int_R^3 V(x) e^-iqx dx"""
```

Comment:
The formula is actually consistent, it's just the function takes q as a number (because it only depends on its magnitude). We could also have a version that takes as input a vector and calls local_potential_fourier(norm(q)), but there's no real point. We need to have this in a theory document somewhere, once we have that it'll be clearer.

Reply:
That's what I meant: It does not make sense when `q` is a number.